ckanext/xloader/tests/test_plugin.py

```diff
@@ -9,6 +9,21 @@
 from six import text_type as str
 from ckan.tests import helpers, factories
 from ckan.logic import _actions
+from ckanext.xloader.plugin import _should_remove_unsupported_resource_from_datastore
+
+
+@pytest.fixture
+def mock_toolkit_config(request):
+    with mock.patch('ckan.plugins.toolkit.config.get') as mock_get:
+        mock_get.return_value = request.param
+        yield mock_get
```

Comment:
This will work, but a simple `return` would be just as good. `yield` is intended for iteratively assembling the return value, which we aren't doing here.